k/kernel-headers-system-docker.yml

```diff
@@ -0,0 +1,9 @@
+kernel-headers-system-docker:
+  image: rancher/os-headers:v0.4.0-dev
+  labels:
+    io.rancher.os.detach: false
+    io.rancher.os.after: network
+    io.rancher.os.scope: system
+  volumes:
+  - /usr/src:/usr/src
```

Comment:
Tried it on os master (in the default console): odd, but /usr/src wouldn't survive reboot. Is that because we recreate /usr for the dev version?
In persistent consoles, it's probably fine not to bind-mount /usr/src, because it will persist as part of the console container filesystem.